inc/shortcodes/class-facebook.php

```diff
@@ -89,6 +89,7 @@ public static function callback( $attrs, $content = '' ) {
 			'#https?://(www)?\.facebook\.com\/video\.php\?v=[\d]+#',
 			'#https?:\/\/www?\.facebook\.com\/+.*?\/videos\/[\d]+\/#',
 			'#https?://(www)?\.facebook\.com\/permalink\.php\?story_fbid=[\d]+&id=[\d]+#',
+			'#https?:\/\/www?\.facebook\.com\/+.*?\/photos\/a\.[\d]+\.[\d]+\.[\d]+\/([\d])+/#',
```

Comment:
@danielbachhuber yeah, that middle bit changes depending on album/position. I'm going with a more general regex to fast forward to the next forward slash and grab the digits after that.